src/components/Search.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) =>  {
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
         placeholder={"Search your Recent Transactions"}
-      />
+        onChange={(e) => props.handleChange(e.target.value)}
```

Comment:
you should probably bind the value of this input to what you get from props as well.